packages/main/cypress/specs/Dialog.cy.ts

```diff
@@ -0,0 +1,413 @@
+import { html } from "lit";
+import "../../src/Dialog.js";
+import "../../src/Button.js";
+import "../../src/Input.js";
+
+describe("Dialog", () => {
+	describe("Dialog general interaction", () => {
+		beforeEach(() => {
+			cy.mount(html`
+				<ui5-button id="btnOpenDialog">Open Stretched Dialog</ui5-button>
+				<ui5-dialog id="dialog" header-text="Dialog">
+					<span>Some content</span>
+					<div slot="footer" class="dialogFooter">
+						<ui5-button id="btnCloseDialog">Close</ui5-button>
+					</div>
+				</ui5-dialog>
+			`);
+		});
+
+		it("tests dialog toggling", () => {
+			cy.get("#btnOpenDialog").then($btnOpenDialog => {
```

Comment:
this test seems like redundant. it is testing an application side logic for open / close